extensions/wikia/FacebookTags/FacebookTagsHooks.php

```diff
@@ -1,14 +1,36 @@
 <?php
 
 class FacebookTagsHooks {
-	public static function onParserFirstCallInit( Parser $parser ): bool {
+	/**
+	 * Hook: ParserFirstCallInit
+	 * Register preprocessor for XFBML widgets in wikitext
+	 *
+	 * @see https://www.mediawiki.org/wiki/Manual:Hooks/ParserFirstCallInit
+	 * @param Parser $parser
+	 */
+	public static function onParserFirstCallInit( Parser $parser ) {
 		foreach ( FacebookTagConstants::SUPPORTED_TAGS as $parserTagName ) {
 			$colonPosition = strpos( $parserTagName, ':' );
 			$htmlTagName = substr( $parserTagName, $colonPosition + 1 );
 
 			$parser->setHook( $parserTagName, new FacebookTagParser( $htmlTagName ) );
 		}
+	}
 
-		return true;
+	/**
+	 * Hook: BeforePageDisplay
+	 * Register JS module which will parse any Facebook widgets on article pages.
+	 * Ideally this module would be added on-demand while parsing XFBML tags,
+	 * but we also have to support HTML widgets added by users.
+	 *
+	 * @see https://www.mediawiki.org/wiki/Manual:Hooks/BeforePageDisplay
+	 * @param OutputPage $out
+	 * @param Skin $skin
+	 */
+	public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$skin". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)